src/migrations/m190821_150226_discount_purchaseTotal_update.php

```diff
@@ -16,7 +16,6 @@ class m190821_150226_discount_purchaseTotal_update extends Migration
     public function safeUp()
     {
         $this->alterColumn('{{%commerce_discounts}}', 'purchaseTotal', $this->decimal(14, 4));
-        true;
```

Comment:
@lukeholder doesn't the `safeUp` method have to `return true` for a successful migration?